docs/src/examples/components/Animation/Types/AnimationExampleKeyframeParams.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { Animation, Icon, Provider } from '@stardust-ui/react'
+
+const colorChanger = {
```

Comment:
it will be great to ensure that types of the `keyframeParams` and input arg of the `keyframe` match each other